### ISSUE TITLE
Support IPython as python console

### ISSUE THREAD

**Issue by declension** [quodlibet/quodlibet#1847]
Ipython would be a much richer environment for exploring songs using Python in QL.

Looks like it could be done according to [their docs on embedding](https://ipython.org/ipython-doc/3/interactive/reference.html#embedding-ipython).

**Comment by declension** [comment]
Much more useful (but old) [Gnome GTK+ IPython integration](https://git.gnome.org/browse/accerciser/tree/plugins/ipython_view.py)...

**Comment by CreamyCookie** [comment]
Yeah, that would be awesome! Just using the plugin, and I'm missing things like:

* Indent by 4 spaces
* Continue indentation
* Syntax highlighting
